content/code-style.md

```diff
@@ -41,7 +41,7 @@ secondStatement();
 
 <h3 id="automatic-error-checking">Automatic error checking</h3>
 
-Having a consistent style means that it's easier to adopt standard tools for error checking. For example, if you adopt a convention that you must always use `let` or `const` instead of `var`, you can now use a tool to ensure all of your variables are scoped the way you expect. That means you can avoid bugs where variables act in unexpected ways. Also, by enforcing that all variables are declared before use, you can easily catch typos before even running any code!
```

Comment:
Removing `easily` here works, but as another option I might have recommended changing it to say "more easily" (e.g. "you can more easily catch typos").

That is to say, as a general rule I think it's acceptable to use _relative_ comparison, so long as we don't call things "easy" without substantiation.

Just to repeat though: I'm not asking for this change, purely pointing out my thoughts; the removal of `easily` works just as well here.

Reply:
agreed